nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
+---
+
+The [distributor](/nservicebus/scalability-and-ha/distributor) has been deprecated and replaced by [sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md) with NServiceBus Version 6.
+
+## Client-side distribution
+
+With sender-side distribution, senders are aware of scaled out receivers, allowing senders to distribute messages directly to the receivers instead of using an intermediary distributor.
+
+
+## Upgrade scenario
+
+This process aims to allow upgrade in production environments without missing events and minimal downtime.
+
+WARNING: To avoid duplicate events, the Distributor node needs to be upgraded to a worker node using NServiceBus 6 (it can be decommissioned after all workers are upgraded to Version 6).
+
+* Upgrade all endpoints communicating to the Distributor (by commands or event subscription messages) to NServiceBus Version 6 first.
```

Comment:
@janovesk - Let me clarify. What I meant is, 

>Upgrade all endpoints communicating to the Distributor (by commands or event subscription messages) to NServiceBus Version 6 first.

In here, not sure if this includes the worker, because technically workers also send messages to the distributor. 

Perhaps clarify it as:

Upgrade all endpoints that interact with the distributor to Version 6 first. For example, upgrade all endpoints that send command messages to the Distributor endpoint for the load to be distributed, or endpoints that send subscription messages to the distributor,  subscribing to events when a worker node publishes an event. At this stage, do not upgrade the workers to Version 6.